x-pack/plugins/monitoring/dev_docs/reference/terminology.md

```diff
@@ -1,14 +1,26 @@
 # Terminology
 
+### Monitored product / Stack component
+
+In some cases these may just be referred to as "products" or "components", these are the individual parts of the Elastic stack that Stack Monitoring, as an application, is concerned with monitoring. Examples include Elasticsearch, Kibana, Logstash, Beats, APM, etc. 
+
+### Collection modes
+
+Stack monitoring data (i.e. the data collected about a given product) has been collected in many different ways over the years. A collection mode is a method for getting these data into Elasticsearch to be stored for later query retrieval.
+
 #### Internal collection
 
-The process of collecting monitoring data handled by the stack components themselves. Each component is responsible for sending documents to elasticsearch directly.
+The process of collecting monitoring data handled by the stack components themselves. Each component is responsible for sending documents to the Elasticsearch monitoring deployment directly.
 
-#### Metricbeat collection
+#### Standalone Beat collection
```

Comment:
There's a link on https://github.com/elastic/kibana/blob/f545deff2329077e01fe0c377cd28a24b71377e9/x-pack/plugins/monitoring/dev_docs/how_to/local_setup.md#L25 that this breaks.

@elastic/obs-docs - anyone know if there's a good way to enable link checking for kibana plugin docs?

Reply:
This also has me noticing that https://github.com/elastic/kibana/blob/c82c6a73fc9bc34268b8f0f7d66041ddf6930397/x-pack/plugins/monitoring/dev_docs/reference/data_collection_modes.md?plain=1#L68 has "Standalone Metricbeat collection" 

Guessing we should maybe update that heading and references as well to "Standalone Beat collection"

Reply:
> @elastic/obs-docs - anyone know if there's a good way to enable link checking for kibana plugin docs?

Hmm. I don't know. I'm going to throw this one at @elastic/kibana-docs. Do you know if this is possible?